client/cmd/job/deploy.go

```diff
@@ -0,0 +1,213 @@
+package job
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/goto/salt/log"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+
+	"github.com/goto/optimus/client/cmd/internal/connection"
+	"github.com/goto/optimus/client/cmd/internal/logger"
+	"github.com/goto/optimus/client/cmd/internal/survey"
+	"github.com/goto/optimus/client/local/model"
+	"github.com/goto/optimus/client/local/specio"
+	"github.com/goto/optimus/config"
+	pb "github.com/goto/optimus/protos/gotocompany/optimus/core/v1beta1"
+)
+
+const (
+	deployTimeout = time.Minute * 30
+)
+
+type deployCommand struct {
+	logger     log.Logger
+	connection connection.Connection
+
+	configFilePath string
+
+	projectName   string
+	namespaceName string
+	host          string
+
+	clientConfig    *config.ClientConfig
+	namespaceSurvey *survey.NamespaceSurvey
+
+	jobNames []string
+}
+
+// NewDeployCommand initializes command for deploying job specifications
+func NewDeployCommand() *cobra.Command {
+	deploy := &deployCommand{
+		clientConfig: &config.ClientConfig{},
+	}
+	cmd := &cobra.Command{
+		Use:     "deploy",
+		Short:   "Deploy optimus job specifications",
+		Long:    "Deploy (create or modify) specific optimus job specifications",
+		Example: "optimus job deploy --jobs [<job_name>,<job_name>] --namespace [namespace_name]",
+		RunE:    deploy.RunE,
+		PreRunE: deploy.PreRunE,
+	}
+	cmd.Flags().StringVarP(&deploy.configFilePath, "config", "c", config.EmptyPath, "File path for client configuration")
+	cmd.Flags().StringVar(&deploy.host, "host", "", "Optimus service endpoint url")
+
+	cmd.Flags().StringVarP(&deploy.namespaceName, "namespace", "n", deploy.namespaceName, "Namespace name in which the job resides")
+	cmd.Flags().StringSliceVarP(&deploy.jobNames, "jobs", "J", nil, "Job names")
+
+	cmd.MarkFlagRequired("jobs")
+	cmd.MarkFlagRequired("namespace")
+	return cmd
+}
+
+func (e *deployCommand) PreRunE(_ *cobra.Command, _ []string) error {
+	if err := e.loadConfig(); err != nil {
+		return err
+	}
+	e.logger = logger.NewClientLogger()
+	e.namespaceSurvey = survey.NewNamespaceSurvey(e.logger)
```

Comment:
if it's not used, we can remove this one

Reply:
removed this